FitGoal/FitGoal/HomeViewController.swift

```diff
@@ -11,42 +11,46 @@ import UIKit
 var imageCache: [URL: UIImage] = [:]
 
 class HomeViewController: UIViewController {
-    lazy var topView: UIView = {
-        let gradientView = GradientView(frame: CGRect(x: 0, y: 0, width: 600, height: 812))
-        gradientView.layer.cornerRadius = 150
-        gradientView.layer.maskedCorners = [.layerMinXMaxYCorner]
-        gradientView.colors = [#colorLiteral(red: 0.2816967666, green: 0.8183022738, blue: 0.9222241044, alpha: 1), #colorLiteral(red: 0.5647058824, green: 0.07450980392, blue: 0.9568627451, alpha: 1)]
-        let rotation = CGAffineTransform(rotationAngle: -26 / 180 * CGFloat.pi)
-        gradientView.transform = rotation.translatedBy(x: 10, y: -600)
+    
+    private var gradientView: UIView = {
+        let gradientView = GradientView()
+        gradientView.layer.cornerRadius = 7
+        gradientView.colors = [#colorLiteral(red: 0.03921568627, green: 0, blue: 0, alpha: 0.27), #colorLiteral(red: 0.9411764706, green: 0.7137254902, blue: 0.7137254902, alpha: 0)]
         return gradientView
     }()
 
-    let suggestionsCollectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+    let homeCollectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
 
     var workoutSuggestions = [Routine]()
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         self.view.backgroundColor = #colorLiteral(red: 0.9647058824, green: 0.9647058824, blue: 0.9647058824, alpha: 1)
-        self.view.addSubview(topView)
-        self.view.addSubview(suggestionsCollectionView)
-
-        self.suggestionsCollectionView.dataSource = self
-        self.suggestionsCollectionView.delegate = self
+        self.view.addSubview(homeCollectionView)
+        self.view.addSubview(gradientView)
+        
+        
+        self.homeCollectionView.dataSource = self
+        self.homeCollectionView.delegate = self
 
-        self.suggestionsCollectionView.register(
+        self.homeCollectionView.register(
             SuggestedRoutineCell.self,
             forCellWithReuseIdentifier: SuggestedRoutineCell.indentifier
         )
         
-        self.suggestionsCollectionView.register(
+        self.homeCollectionView.register(
             RoutineSectionHeader.self,
             forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader,
             withReuseIdentifier: RoutineSectionHeader.identifier
         )
+        self.homeCollectionView.register(
+            GoalTrakerCell.self,
+            forCellWithReuseIdentifier: GoalTrakerCell.identifier
+        )
+        
 
-        self.suggestionsCollectionView.alwaysBounceVertical = true
-        self.suggestionsCollectionView.backgroundColor = .none
+        self.homeCollectionView.alwaysBounceVertical = true
+        self.homeCollectionView.backgroundColor = .none
```

Comment:
It looks like you don't want to use `.none` here. If what you want is a transparent color, what you need is `.clear`. Using `.none` is equivalent to using `nil`, which mean that you won't set any background color, which mean that the collection view internal implementation will decide what color to use (probably something like transparent, black or white).

The reason why `.none` is equivalent to using `nil` is because `nil` is actually a syntactical sugar (i.e handy way) provided by the compiler to use optional values in the code. An optional type is defined as an enum value:

```swift
enum Optional<Value> {
    case some(Value)
    case none
}
```

But to improve code readability the compiler treats that type in a special way, allowing you to write _nicer_ code. Consider the following cases with their respective _real_ versions:

```swift
var age: Int? = 5
// var age: Optional<Int> = .some(5)

age = 3
// age = .some(3)

age = nil
// age = .none

if let number = age {
    print(number)
}
else {
    print("Value not set")
}

//  switch age {
//  case .some(number):
//      print(number)
//  case .none:
//      print("Value not set")
```

Usually you will use the _nicer_ way to handle optionals, but the _real_ way is still valid so you can use that version as well. That's precisely what you are doing when you set `backgroundColor = .none`.